File: submissions/CodesNil/problem3.java
import java.io.*;
class problem3
{
public int prime(int r)
{
int l=0;
int g=0;
for(int i=1;i<=r;i++)
{
l=r%i;
if(l==0)
g=g+1;
}
return g;
}
public static void main(String args[])throws IOException
{
int sum=0;
int r=0;
problem3 ob=new problem3();
BufferedReader BR=new BufferedReader(new InputStreamReader(System.in));
int n=Integer.parseInt(BR.readLine());
int nn=n;
while(nn!=0)
{
r=nn%10;
sum=sum*10+r;
nn=nn/10;
}
if(ob.prime(n)==2 && sum==n)
System.out.println("Palprime");
else
System.out.println("Not Palprime");
}
}

File: submissions/sagar_sandeep/problem3.cpp
#include<stdio.h>
int prime(int n)
{
	int i,c=0;
	for(i=2;i<n;i++)
	{
		if(n%i==0)
		c++;
	
	}
	if(c==0)
	return 1;
	else
	return 0;
}
int pal(int n)
{
	int m=n,d=0,r=0;
	while(m>0)
	{
		d=n%10;
		r=r*10+d;
		n/=10;
	}
	if(r==m)
	return 1;
	else
	return 0;
}
int main()
{
	int n;
    int m=n;
	scanf("%d",&n);
	if((pal(n)==1)&&(prime(m)==1))
	printf("Palprime");
	else
	printf("Not Palprime");
	return 1;
}
	
